staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -46,7 +47,7 @@ import (
 func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Interface, includeName bool) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
 		// For performance tracking purposes.
-		trace := utiltrace.New("Create", utiltrace.Field{"url", req.URL.Path})
+		trace := utiltrace.New("Create", utiltrace.Field{Key: "url", Value: req.URL.Path}, utiltrace.Field{Key: "user-agent", Value: utilnet.MaybeTruncateUserAgent(req)}, utiltrace.Field{Key: "client", Value: &utilnet.LazyClientIp{}})
```

Comment:
don't all of these need the req to be included in LazyClientIP?
```suggestion
		trace := utiltrace.New("Create", utiltrace.Field{Key: "url", Value: req.URL.Path}, utiltrace.Field{Key: "user-agent", Value: utilnet.MaybeTruncateUserAgent(req)}, utiltrace.Field{Key: "client", Value: &utilnet.LazyClientIp{req}})
```